src/databricks/labs/ucx/mixins/fixtures.py

```diff
@@ -1073,6 +1087,11 @@ def create(  # pylint: disable=too-many-locals,too-many-arguments,too-many-state
             storage_location = storage_override
 
         sql_backend.execute(ddl)
+
+        # CTAS AND NON_DELTA does not support TBLPROPERTIES
+        if ctas or non_delta:
+            sql_backend.execute(ctas_tbl_properties_ddl)
```

Comment:
Execute a second sql query that sets table properties

Reply:
Yup the query being executed here 'ctas_tbl_properties_ddl' is the second ALTER TABLE query for setting table properties

Reply:
Renamed it to 'alter_table_tbl_properties_ddl'